manage/app/routes/campaign/create.js

```diff
@@ -1,23 +1,27 @@
 import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
 import { RouteQueryManager } from 'ember-apollo-client';
 import FormMixin from 'leads-manage/mixins/form-mixin';
 import { get } from '@ember/object';
 
 import mutation from 'leads-manage/gql/mutations/campaign/create';
 
 export default Route.extend(RouteQueryManager, FormMixin, {
+  config: inject(),
   model() {
     return {
       maxIdentities: 200,
       range: {
         start: null,
         end: null,
       },
+      showAdvertiserCTOR: typeof this.config.isSettingSet("advertiserCTORInitialVisibility") === 'boolean' ? this.config.isSettingSet("advertiserCTORInitialVisibility") : true,
+      showTotalAdClicksPerDay: typeof this.config.isSettingSet("totalAdClicksPerDayInitialVisibility") === 'boolean' ? this.config.isSettingSet("totalAdClicksPerDayInitialVisibility") : true,
```

Comment:
Update your method in the config class to be `getSetting` and have it accept a default value. Return the setting if the value is set (aka your `find` returns something that isn't null), or else the supplied default value. Something like this:
```js
getSetting(key, defaultValue = false) {
  const found = this.settings.find((s) => s.key === key);
  return found ? found.value : defaultValue;
}
```

Then this can change to be simpler here:
```suggestion
      showAdvertiserCTOR: this.config.getSetting('advertiserCTORInitialVisibility', true),
      showTotalAdClicksPerDay: this.config.getSetting('totalAdClicksPerDayInitialVisibility', true),
```

Always (except for string interpolation) use single quotes in JS files -- double quotes should only be used in templates. The linter _should_ have complained about that here, not sure why it didn't.